Makefile

```diff
@@ -21,7 +21,7 @@ endif
 TEST_FORMAT = short-verbose
 endif
 
-PIPELINE_VERSION = 0.61.0
+PIPELINE_VERSION = sa-role-velero
```

Comment:
**Reminder**: don't forget to update the Pipeline version to a valid tag once available.